MarketMatrix/src/components/HomeScreenComponents/TotalValueLineChart.tsx

```diff
@@ -0,0 +1,78 @@
+import {ChartConfiguration} from '../chart/ChartConfiguration';
+import {useEffect, useState} from 'react';
+import {StockChanges, UserData} from '../../constants/Interfaces';
+import axios from 'axios';
+
+interface TotalValueLineChartProps {
+  currency: string;
+}
+
+export const TotalValueLineChart = ({currency}: TotalValueLineChartProps) => {
+  const [user, setUser] = useState<UserData[]>([]);
+  const [changes, setChanges] = useState<StockChanges[]>([]);
+
+  let chartValues: number[] = [];
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await axios.get('http://localhost:8081/src/db.json');
```

Comment:
You can create a separate function and move it to http directory and then use it here